drivers/hmis/spec/requests/hmis/project_enrollment_search_spec.rb

```diff
@@ -86,6 +86,66 @@
       end
     end
   end
+
+  describe 'project enrollments query' do
+    let(:query) do
+      <<~GRAPHQL
+        query TestQuery($id: ID!, $filters: EnrollmentsForProjectFilterOptions!) {
+          project(id: $id) {
+            id
+            enrollments(limit: 10, offset: 0, filters: $filters) {
+              nodesCount
+              nodes {
+                id
+              }
+            }
+          }
+        }
+      GRAPHQL
+    end
+
+    describe 'household tasks filter' do
```

Comment:
The tests only cover single-person households. Can you add some where members are in the same households?
That would look like:
```
        # Entered a year ago, annual is due
        e4 = create(:hmis_hud_enrollment, data_source: ds1, project: p1, client: c3, entry_date: Date.today - 1.year)
        # Entered less than a year ago, annual is not due
        create(:hmis_hud_enrollment, data_source: ds1, project: p1, client: c3, entry_date: Date.today - 6.months, household_id: e4.household_id)
```

If there is a 3-person household where only 2 members have completed annuals, the "Annuals Due" filter on the Enrollment table should only include the 1 member that is missing an annual. I think it works this way based on reading the query, but we should have tests to back it. Thanks!

Reply:
Done